### ISSUE TITLE
Dashboard A11y: Button - focus style 

### ISSUE THREAD

**Issue by BrittanyIRL** [GoogleForCreators/web-stories-wp#4196]
## Bug Description
The buttons with the blue background in the dashboard are really hard to tell when they are focused. These include: 
- Create New Story (side nav) 
- Use Template (detail template view) 

## Expected Behaviour
Focus outline should be distinguishable. 
These button styles are getting changed with the design system. Updating to the new designs will fix this. 

## Additional Context

- Plugin Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance Criteria

Ensure the focus is distinguishable

## Implementation Brief

Either update the button to fit the new design specs or adjust the existing button so the blue outline is distinguishable in the dashboard.

**Comment by csossi** [comment]
Verified in QA
